webpack-and-babel/imports.js

```diff
@@ -0,0 +1,22 @@
+/* eslint-disable no-unused-vars */
+
+// import from a file located at the absolute path `/foo/bar/baz/qux.js`:
+// eslint-disable-next-line import/no-absolute-path
+import f1 from '/foo/bar/baz/qux';
+
+// import from a file named `foo.json` located in the current directory:
+import f2 from './foo';
```

Comment:
This still needs `.json`. Webpack will try and add file extensions automatically for `.js`, `.jsx`, `.ts`, `.tsx`, but not for `.json`. So we still need to explicitly state that we want a `.json` file